tests/scripts/thread-cert/border_router/test_plat_udp_accessiblity.py

```diff
@@ -126,7 +126,7 @@ def _test_srp_server(self, client, server_addr, server_port):
         # check if the SRP client can register to the SRP server
         client.srp_client_start(server_addr, server_port)
         client.srp_client_set_host_name('host1')
-        client.srp_client_set_host_address(client.get_rloc())
```

Comment:
Should also be able to register the service using RLOC ?

Reply:
Yes. I've updated the advertising proxy PR so this one is not needed.